docker-ci.sh

```diff
@@ -44,6 +44,7 @@ function run_test {
     [ -n "$CONSTEL" ] && url="$url&constellation=$CONSTEL"
     [ -n "$MOBILE_UA" ] && url="$url&deviceMode=true"
     [ -z "$JQUERY"  ] && url="$url&nojquery=true"
+    [ -n "$PERF" ] && url="$url&include=DevExpress.performance&workerInWindow=true&jquery="
```

Comment:
Why is `jquery=` instead of `jquery=true`?

Reply:
This is an extra parameter. JQuery is enabled by default.
Got rid of it.